platform/linux-generic/odp_packet.c

```diff
@@ -1380,7 +1342,7 @@ odp_packet_t odp_packet_copy(odp_packet_t pkt, odp_pool_t pool)
 	odp_packet_t newpkt = odp_packet_alloc(pool, pktlen);
 
 	if (newpkt != ODP_PACKET_INVALID) {
-		if (_odp_packet_copy_md_to_packet(pkt, newpkt) ||
+		if (_odp_packet_copy_md(srchdr, packet_hdr(newpkt), 1) ||
```

Comment:
copy_md() return value can be removed if caller checks uarea sizes beforehand. Optimally uarea sizes would be checked from src vs dst pool, so that packet_alloc() is avoided if uarea sizes don't match.

Reply:
Fixed in V2.